components/device-mgt/org.wso2.carbon.device.mgt.common/src/main/java/org/wso2/carbon/device/mgt/common/DeviceOrganizationTree.java

```diff
@@ -0,0 +1,5 @@
+package org.wso2.carbon.device.mgt.common;
```

Comment:
What is the purpose of having this empty class?

Reply:
It was a redundant class used for an unimplemented feature. It has been removed